### ISSUE TITLE
Support the first line of pipeless-text being indented

### ISSUE THREAD

**Issue by ForbesLindesay** [jadejs/jade#1588]
e.g.

```jade
//
    .foo
  .bar
```

or

```jade
:markdown
      code sample

  # Heading
```

or

```js
pre.
    what
  is going on
```

[see #1583]

At the moment these all result in the error "Unexpected token 'pipless-text'".  They used to render to something un-expected.  It's time we fixed this.

**Comment by ForbesLindesay** [comment]
A blank line should also be allowed, e.g.

h1.page-header OAuth 2

```jade
:markdown

  ### Description
```

**Comment by TimothyGu** [comment]
Fixed in jadejs/jade-lexer#16.

**Comment by tomByrer** [comment]
Has this been released to the public yet?  Would have saved me 1/2hour of debugging...
